aosp_diff/linux_trusty/system/core/0025-Remove-or-modify-keystore-APIs-for-Clear-Linux.patch

```diff
@@ -0,0 +1,71 @@
+From c095e775ee7bfe282278b0adeb068edd095cf5c6 Mon Sep 17 00:00:00 2001
+From: Chen Gang G <gang.g.chen@intel.com>
+Date: Thu, 11 Apr 2019 16:07:00 +0800
+Subject: [PATCH] Remove or modify keystore APIs for Clear Linux
+
+Some API is not available on clear linux
+
+Signed-off-by: Chen Gang G <gang.g.chen@intel.com>
+---
+ trusty/keymaster/legacy/trusty_keymaster_device.cpp | 12 ++++++++++++
+ 1 file changed, 12 insertions(+)
+
+diff --git a/trusty/keymaster/legacy/trusty_keymaster_device.cpp b/trusty/keymaster/legacy/trusty_keymaster_device.cpp
+index 13db10f..a5c8e9a 100644
+--- a/trusty/keymaster/legacy/trusty_keymaster_device.cpp
++++ b/trusty/keymaster/legacy/trusty_keymaster_device.cpp
+@@ -146,11 +146,15 @@ keymaster_error_t TrustyKeymasterDevice::configure(const keymaster_key_param_set
+ 
+     AuthorizationSet params_copy(*params);
+     ConfigureRequest request(message_version_);
++#if 0
+     if (!params_copy.GetTagValue(TAG_OS_VERSION, &request.os_version) ||
+         !params_copy.GetTagValue(TAG_OS_PATCHLEVEL, &request.os_patchlevel)) {
+         ALOGD("Configuration parameters must contain OS version and patch level");
+         return KM_ERROR_INVALID_ARGUMENT;
+     }
++#endif
++    request.os_version = 0;
```

Comment:
could you add some comments for this change?

Reply:
updated.